src/ray/stats/metric_defs.h

```diff
@@ -46,14 +46,14 @@ DECLARE_stats(grpc_server_req_handling);
 DECLARE_stats(grpc_server_req_finished);
 
 /// GCS Resource Manager
-DECLARE_stats(new_resource_creation_latency_ms);
+DECLARE_stats(gcs_new_resource_creation_latency_ms);
 
 /// Placement Group
-DECLARE_stats(placement_group_creation_latency_ms);
-DECLARE_stats(placement_group_scheduling_latency_ms);
-DECLARE_stats(pending_placement_group);
-DECLARE_stats(registered_placement_group);
-DECLARE_stats(infeasible_placement_group);
+DECLARE_stats(gcs_placement_group_creation_latency_ms);
+DECLARE_stats(gcs_placement_group_scheduling_latency_ms);
+DECLARE_stats(gcs_num_placement_group);
```

Comment:
```suggestion
DECLARE_stats(gcs_num_placement_groups);
```

Reply:
Consider instead `gcs_placement_group_count` since that gives all gcs_placement_group_* metrics a uniform prefix.